tests/features/logIn.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { test, expect, Page } from '@playwright/test';
+import LoginPage from '../poms/LoginPage/LoginPage';
+import MainDashboard from '../poms/MainDashboard/MainDashboard';
+import { userName, userPwd } from '../data';
+
+
+test.describe('Log in test suite', () => {
+    test('Log in to the app', async ({page}) => {
+        const loginPage = new LoginPage(page);
+        const mainDashboard = new MainDashboard(page);
+
+        await loginPage.goTo();
+        await loginPage.pageLoaded();
```

Comment:
You do this twice. The same method is hidden in goTo()

Reply:
fixed